Configuration/TypoScript/Compatibility/constants.typoscript

```diff
@@ -0,0 +1,21 @@
+#####################################
+### PAGE compatibility for T3 < 9 ###
+#####################################
+page {
+    theme {
```

Comment:
i think this should be moved one level up to `page.language` or even move to `config.language` because this is a global setting and not "theme" specific.

Reply:
mmh, currently i don't think it will be used for the core because there we always should take the defaults from sites ext. but moving up makes sense.

Reply:
talked to @psychomieze on sunday about this, current state is that sites will be optional for the beginning. so we could also make this available for cms9 when sites are not used.